observability/power_monitoring/power-monitoring-release-notes.adoc

```diff
@@ -15,6 +15,35 @@ These release notes track the development of {PM-title} in the {product-title}.
 
 For an overview of the {PM-operator}, see xref:../power_monitoring/power-monitoring-overview.adoc#power-monitoring-about-power-monitoring_power-monitoring-overview[About {PM-shortname}].
 
+[id="power-monitoring-release-notes-0-3"]
+== {PM-shortname-c} 0.3 (Technology Preview)
+
+This release includes the following version updates:
+
+* {PM-kepler} 0.7.12
+* {PM-operator} 0.15.0
+
+The following advisory is available for {PM-shortname} 0.3:
+
+* link:https://access.redhat.com/errata/RHEA-2024:9961[RHEA-2024:9961]
+
+[id="power-monitoring-bug-fix-0-3-features"]
+=== Bug fix
```

Comment:
```suggestion
=== Bug fixes
```

Reply:
@xenolinux Done! :) Thank you.

Reply:
Thanks. I will merge the PR once builds pass.